### DISCUSSION TITLE
modify project pages

### DISCUSSION THREAD

**Discussion by kdhenderson** [alshedivat/al-folio#2294] [Q&A] [answered]
I am new to using jekyll themes and and have only modified simple html website templates in the past. I'd like to take a Rmd file and possibly paste the knitted html in one of the projects pages. I can successfully do this, but when I do it I run into some undesirable formatting issues. The font size of the navigation bar, project title, project description and footer are all much smaller than the html that I paste in. I also mess up the navigation bar so it no longer always appears and the purple line across the bottom then runs over the html I pasted when the page is displayed. Maybe someone can give me some tips. I'm finding that making modifications is more complex than I originally realized. Thanks.

**Comment by george-gca** [comment, marked as answer]
I'd recommend you to take a look at the [CUSTOMIZE](https://github.com/alshedivat/al-folio/blob/master/CUSTOMIZE.md) part of our README and also [jekyll official documentation](https://jekyllrb.com/docs/pages/). Check the directory structure to familiarize with it, and also how jekyll works.
